jetstream/outcomes/firefox_desktop/newtab_do_no_harm.toml

```diff
@@ -17,3 +17,23 @@ Guardrail metrics for the New Tab. Includes user experience and revenue-related
 [metrics.newtab_engagement.statistics.binomial]
 [metrics.newtab_any_searches.statistics.binomial]
 [metrics.newtab_gt4_searches.statistics.binomial]
+
+[metrics.sponsored_tile_clicks.statistics.sum]
```

Comment:
the `sum` is kinda a weird statistic, it sums the client-level values into a total per branch, but doesn't have confidence intervals or anything. If you have any interest in looking at these middle metrics, you might want to also add one of the `_mean` statistics so that you get confidence intervals